multicluster/charts/linkerd-multicluster-link/templates/service-mirror-policy.yaml

```diff
@@ -0,0 +1,65 @@
+---
+apiVersion: policy.linkerd.io/v1alpha1
+kind: Server
+metadata:
+  namespace: {{.Values.namespace}}
+  name: service-mirror
+  labels:
+    linkerd.io/control-plane-component: linkerd-service-mirror
+    mirror.linkerd.io/cluster-name: {{.Values.targetClusterName}}
+spec:
+  podSelector:
+    matchLabels:
+      linkerd.io/control-plane-component: linkerd-service-mirror
+      mirror.linkerd.io/cluster-name: {{.Values.targetClusterName}}
+  port: admin-http
+  proxyProtocol: HTTP/1
+---
+apiVersion: policy.linkerd.io/v1alpha1
+kind: ServerAuthorization
+metadata:
+  namespace: {{.Values.namespace}}
+  name: service-mirror
+  labels:
+    linkerd.io/control-plane-component: linkerd-service-mirror
+    mirror.linkerd.io/cluster-name: {{.Values.targetClusterName}}
+spec:
+  server:
+    name: service-mirror
+  client:
+    # Allow prometheus scraping
```

Comment:
can this be restricted to the prometheus service account?

Reply:
I kinda didn't want to add a forced dependency on viz here so I left it open. But now that I think about it, it does make sense to hard-code it to viz's prom because this scraping goal is for `linkerd mc gateways` to work (which already depends on viz). If people want to give access to their own proms, they should just create a separate saz on top :+1: